query/src/pipelines/transforms/group_by/aggregator.rs

```diff
@@ -112,12 +112,12 @@ impl<Method: HashMethod + PolymorphicKeysHelper<Method>> Aggregator<Method> {
             {
                 for group_key in group_keys.iter() {
                     let mut inserted = true;
```

Comment:
since inserted is override in entity, so declare the inserted outside the loop is resonable?

Reply:
This is a mutable reference, which we declare here and overwrite internally